docs/tutorials/materialization.md

```diff
@@ -47,14 +47,23 @@ using
 all versions using
 [client.materialize.get_versions_metadata()]({{ client_api_paths.materialize }}.get_versions_metadata).
 
-To change the default version, alter the .version property of the
-materialization client.
+To change the default version, alter the `.version` property of the
```

Comment:
@ceesem any objection to this being the preferred pattern advertised in the tutorial now?